bundles/io/org.eclipse.smarthome.io.audio/src/main/java/org/eclipse/smarthome/io/audio/AudioFormat.java

```diff
@@ -0,0 +1,163 @@
+/**
+ * Copyright (c) 2015-2016 Harald Kuhn
+ * 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.io.audio;
+
+/**
+ * An audio format definition 
+ *
+ * @author Harald Kuhn (hkuhn42) initial api
+ */
+public class AudioFormat {
+
+    /**
+     * Codec
+     */
+    private String codec;
+
+    /**
+     * Container
+     */
+    private String container;
+
+    /**
+     * Big endian or little endian
+     */
+    private boolean bigEndian;
```

Comment:
I guess this comment is making it clear that you are have two use cases
for an `AudioFormat` in mind.

1. An `AudioFormat` can be used to specify an exact container, codec...
2. An `AudioFormat` can be used to specify a set of exact `AudioFormat`'s

For the first use case `bigEndian` can be a `boolean` but for the second use case
`bigEndian` must be a `Boolean`, as setting `bigEndian` to `null` is equivalent to
specifying the set of exact `AudioFormat`'s where `bigEndian` is `true` or `false`. 

As far as I recall we never discussed this in #584, but it seems a reasonable addition